azure/services/virtualnetworks/virtualnetworks.go

```diff
@@ -95,9 +108,15 @@ func (s *Service) Reconcile(ctx context.Context) error {
 				AddressSpace: &network.AddressSpace{
 					AddressPrefixes: &vnetSpec.CIDRs,
 				},
+				// 	VirtualNetworkPeerings: &peerings,
```

Comment:
delete?

I'm going to stop adding these comments. Please remove commented code.